ibmvmc/ibmvmc.rst

```diff
@@ -0,0 +1,252 @@
+.. SPDX-License-Identifier: GPL-2.0+
+======================================================
+IBM Virtual Management Channgel Kernel Driver (IBMVMC)
+======================================================
+
+:Authors:
+	Dave Engebretsen <engebret@us.ibm.com>,
+	Adam Reznechek <adreznec@linux.vnet.ibm.com>,
+	Steven Royer <seroyer@linux.vnet.ibm.com>,
+	Bryant G. Ly <bryantly@linux.vnet.ibm.com>,
+
+Introduction
+============
+
+Note: Knowledge of virtualization technology is required to understand
+this document.
+
+The Virtual Management Channel (VMC) is a logical device which provides an
+interface between the hypervisor and a management partition. This interface
+is like a message passing interface. VMC also allows for direct hypervisor
+configuration without requiring more network connections. This management
+partition is intended to provide an alternative to HMC-based system
+management.
+
+HMC
+---
+
+The primary hardware management solution that is developed by IBM relies
```

Comment:
Maybe don't talk about IBM or IBM specific products as much (hard, I know).  You can define an external Hardware Management Console for management.  And a management partition, and a management application (probably don't need to talk about NovaLink directly).